src/components/SummaryList.jsx

```diff
@@ -1,38 +1,39 @@
 import { useState, useEffect } from 'react';
+import SummaryCard from './SummaryCard.jsx';
 
-const SummaryList = () => {
+const SummaryList = ({ topic }) => {
     const [summaryList, setSummaryList] = useState([]);
     const [isLoading, setIsLoading] = useState(true);
 
+    let path = 'https://neilb-nc-news-server.herokuapp.com/api/articles';
+
+    if (topic) {
```

Comment:
If you wanted you could use Axios and the Axios params object. This way you wouldn't have to build the url yourself :)